### ISSUE TITLE
iterator_to_array(new X)[0] invalid quick-fix

### ISSUE THREAD

**Issue by someniatko** [kalessil/phpinspectionsea#1303]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate) v2.0.15 |
| Language level | PHP 7.2 |

Current behaviour
--
Consider the following snippet:

```php
<?php

class Collection implements Iterator {
    public function current()
    {
    }

    public function next()
    {
    }

    public function key()
    {
    }

    public function valid()
    {
    }

    public function rewind()
    {
    }
}

iterator_to_array(new Collection)[0];
```

At the last line, an inspection "Consider using '$collection->current()' instead" suggestion shows up. It also has a suggestion to automatically replace the contents of the line. When i hit Alt+Enter and choose "Use '...->current()' instead", the result line is:
```php
new Collection;
```

Expected behaviour
--
The resulting line is 
```php
(new Collection)->current();
```

Environment details
---
PhpStorm 2019.1.1
Build #PS-191.6707.66, built on April 17, 2019
Licensed to <me>
Subscription is active until <date>
JRE: 1.8.0_202-release-1483-b44 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0

**Comment by kalessil** [comment]
Fixed!
